pythia/modules/metrics.py

```diff
@@ -202,6 +202,59 @@ def calculate(self, sample_list, model_output, *args, **kwargs):
         return value
 
 
+@registry.register_metric("caption_bleu4")
+class CaptionBleu4Metric(BaseMetric):
```

Comment:
Are we providing support for other metrics? SPICE and ROGUE?

Reply:
I have requested a separate PR #78 which adds a script for calculating metrics. If we are ok with importing the dependent libraries for calculating these metrics then we can add support for all of them in `metrics.py`.